.github/workflows/ui-ci.yml

```diff
@@ -0,0 +1,27 @@
+name: UI CI
+
+on:
+  push:
+    branches: [ development ]
+    paths-ignore: [ '*.md']
+  pull_request:
+    branches: [ '**' ]
+  merge_group:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build
+        run: npm --workspaces run build
```

Comment:
Scripts are run in the order in which workspaces are listed in package.json, so this ensures that ui-core is built before its dependents are.